cockroachcloud/export-metrics.md

```diff
@@ -187,7 +187,7 @@ To enable metrics export for your {{ site.data.products.dedicated }} cluster to
     {% include_cached copy-clipboard.html %}
     ~~~shell
     curl --request POST \
-      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/datadog \
       --header "Authorization: Bearer {secret_key}" \
       --data '{"datadog":{"site": "{datadog_site}", "api_key": "{datadog_api_key}}"}}'
```

Comment:
The "datadog" config is unnested now, so this line can be replaced with `--data '{"site": "{datadog_site}", "api_key": "{datadog_api_key}"}'`